src/Components/Onboarding/Steps/Artists/ArtistSearchResults.tsx

```diff
@@ -35,6 +37,10 @@ class ArtistSearchResultsContent extends React.Component<RelayProps, null> {
     )
 
     popularArtistsRootField.setLinkedRecords(updatedPopularArtists, "match_artist", { term: this.props.term })
+
+    this.followCount = this.followCount + 1
```

Comment:
could be:

```ts
this.followCount += 1
```

Reply:
lol I brought this up with @alloy when first implementing this. I don't have strong opinions so I'll go ahead make the change in my next commit.